### PR TITLE
WIP: Applying admin role check for user requests to admin site pages

### PR DESCRIPTION
**Pull request by willh**

### CODE REVIEW

File: src/identity/identity.ts [hunk 1]
```diff
@@ -34,6 +34,16 @@ export class Identity {
 		return this.roles && this.roles.some(value => roles.indexOf(value) > -1)
 	}
 
+	hasAnyAdminRole() {
```
Comment (mmackiewicz1) [inline src/identity/identity.ts:37]: I noticed that this method returns hard coded data. Is there a way to fetch admin roles from identity service via API call? Thanks to this we would not have to update lpg-management code when we update roles. Unless of course the system has been built around hard-coded roles in the first place.

Comment (willh) [reply src/identity/identity.ts:37]: True, but see the rest of the file - we've relied on the app having a specific knowledge of these roles the whole time. If we retrieved them from the DB we'd also need some logic to determine which ones from the DB are 'admin' types and which aren't, which you could push into the DB as a property but then if you wanted to go finer-grained you'd need to start recording in the DB which roles correspond to visibility of which screens/actions so you're starting to push a lot of presentation detail into the data. If you went all the way with that you might be able to keep the app almost free of references to the actual role names but the scope of this change is trying to be consistent with what we've got for the other role checks.

Comment (mmackiewicz1) [reply src/identity/identity.ts:37]: Ok

**Review by mmackiewicz1 — APPROVED** [review]

**Comment by willh** [comment]
Merging hotfix

**Pull request merged**